### ISSUE TITLE
Enchanced steam Mozilla Firefox

### ISSUE THREAD

**Issue by Dicur3x** [jshackles/Enhanced_Steam#1154]
I very love your extension enchanced steam and i very want to support you a little with money, but... i can't. Cause i use Firefox browser and i still can't install it (tried automatic install from site - error, and manual install - error again). I'm use last stable version Firefox, so i don't know why it dont work for me. Maybe i do anything wrong?

**Comment by jshackles** [comment]
What error message are you getting when downloading it from the site?  If you could provide screenshots, that might help.

The manual installation requires [Firefox Developer Edition](https://www.mozilla.org/en-US/firefox/developer/) as the uncompiled code can't be digitally signed by Mozilla.

**Comment by Dicur3x** [comment]
Yes, of course. Here screenshot.
№1
![2016-09-30_225006](https://cloud.githubusercontent.com/assets/13305674/19004805/4726dd1c-8760-11e6-856e-c9bc4ab38730.png)

Just problems with connection (i don't know why). In translate seems like "error while try to make secure connection). So, i can't install it anyway...





Also, a little offtopic, but maybe important: in this moment site www.enhancedsteam.com opens with problems, here screenshots:
№2
![2016-09-30_225117](https://cloud.githubusercontent.com/assets/13305674/19004838/6e546daa-8760-11e6-8e84-fc98e274c515.png)
№3
![2016-09-30_225607](https://cloud.githubusercontent.com/assets/13305674/19005010/2118ff8c-8761-11e6-8ba7-d88503719d14.png)


Screenshot №2- this error appear sometimes, but in other times site looks similiar screenshot №3. Seems like data loading not correct.
Last time i tried it 28 september, it was ok.

**Comment by jshackles** [comment]
Do you get similar error messages when trying to browse the website with any browser other than Firefox?  For example, will it load using Internet Explorer or Chrome?

**Comment by Dicur3x** [comment]
No. I tried it with google chrome and all seems ok, both site and main problem with error while installing extension (chrome installing it good). Opera loads site good too, but extension there have old version 7.1.1. So, now problem only in FF (extension, and now with site).

So, maybe if i switch off some security settings, download extension and switch off, it will be work?

**Comment by jshackles** [comment]
If you can get the site working in other browsers, you should be able to download the compiled and signed extension from there:

https://firefox.enhancedsteam.com/EnhancedSteam.xpi

Then drag this file into Firefox and it should install.  I'm not sure why this isn't working for you, the Firefox extension has been installed regularly through the browser by hundreds of people in the past 24 hours, and I've tested it from three geographically diverse locations as well and was able to install without any issues.

It appears from your screenshots that the issue has to do with accessing the site over SSL, so it's possible that SSL may be blocked on your local firewall or somewhere in your browser settings.

**Comment by Dicur3x** [comment]
When i'm trying to install from that link, i see same error from screenshot №1 and i'm tried even to switch off antivirus - don't work T_T And browser settings in default positions, so, and error says "maybe it error your network", but settings in default, i don't know :( Old versions worked good. And i haven't any problems with browser and SSL, only this extension... idk why.

**Comment by jshackles** [comment]
Can you download the extension directly without using SSL?

http://firefox.enhancedsteam.com/EnhancedSteam.xpi

You'll likely need to do this with a non-Firefox browser, then drag the XPI file into Firefox, because Firefox requires addons loaded through websites to use SSL.

**Comment by Dicur3x** [comment]
It really was big plank in ass for me, and i finally got it with opera (google chrome don't download it too).
So, at least i installed it and i hope, next version will be wirk properly (cause now it beta and working only in first launch on main page of steam store). But when i can use site, i will help you a little for your good extension!

**Comment by JoiNNN** [comment]
I don't recognize the language, so excuse my ignorance if I'm wrong as I'll assume is Russian and let you know that there some Russians ISPs which block Cloudflare IPs, which ES uses for it domains AFAIK. Chances are, it might be because of this why you get blank pages and SSL errors, and there might be other errors with ES in other browsers also.
I recommend you install a free VPN app and give a try, see if the problems persist.

**Comment by VanityFox** [comment]
So, Enhanced Steam is discontinued. There is now [Augmented Steam](https://github.com/tfedor/AugmentedSteam/). Perhaps your requested feature/bugfix is already implemented there! If not, you could move your issue there.
